src/Hocon.Configuration/Config.cs

```diff
@@ -89,7 +81,7 @@ public Config(HoconRoot root) : base(root?.Value, root?.Substitutions ?? Enumera
         /// <remarks>
         ///     Added for brevity and API backwards-compatibility with Akka.Hocon.
         /// </remarks>
-        public static Config Empty => ConfigurationFactory.Empty;
+        public static Config Empty => CreateEmpty();
```

Comment:
Need to make this `readonly`

Reply:
Oh, we don't want it to be read only, its a static property. Since the old Hocon are not immutable, we have to rebuild it every time it is accessed.

Reply:
Yep, my mistake.